packages/components/src/confirm/component.tsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+// eslint-disable-next-line no-restricted-imports
+import type { Ref } from 'react';
+import { Card, CardHeader, CardFooter } from '../card';
+import { Button } from '../button';
+import { Heading } from '../heading';
+
+/**
+ * Internal dependencies
+ */
+import type { OwnProps } from './types';
+import { useConfirm } from './hook';
+import { contextConnect, PolymorphicComponentProps } from '../ui/context';
+
+function Confirm(
+	props: PolymorphicComponentProps< OwnProps, 'div' >,
+	forwardedRef: Ref< any >
+) {
+	const { role, wrapperClassName, ...otherProps } = useConfirm( props );
+
+	return (
+		<div { ...otherProps } role={ role } className={ wrapperClassName }>
+			<Card ref={ forwardedRef }>
+				<CardHeader>
+					<Heading level="4">Are you sure?</Heading>
+				</CardHeader>
+				<CardFooter justify="center">
+					<Button>OK</Button>
```

Comment:
Leaving this here results in a `Cannot add property current, object is not extensible` that looks related to the `Ref` and the React clean-up code around it. It seem the `Button`'s ref is an empty object that has `Object.preventExtensions` applied? That's just a wild guess. Or I might just be missing something in the G2 API? 🤔 

If you remove this `Button`, the component will render in StoryBook.

Reply:
This has been solved. `Button` is default-exported, but I was actually using a named import.